packages/components/src/List/DisplayLarge/DisplayLarge.component.js

```diff
@@ -79,14 +81,14 @@ function rowRenderer({ id, index, columns, item, itemProps, titleProps }) {
 			</div>
 			<div className={theme.columns}>
 				{info.map((group, i) => (
-					<ul key={i}>
+					<dl key={i}>
 						{group.map((obj, j) => (
-							<li key={j}>
+							<dd key={j}>
 								<span className={theme.label}>{obj.label}</span>
 								<span className={theme.value}>{obj.value}</span>
```

Comment:
I'm not sure this is more readable, and if you do that renderObjKey is a React pure component, so you also have to add proptypes, tests, ...